src/styles.ts

```diff
@@ -190,4 +190,7 @@ export const styles = css`
 		width: var(--width);
 		min-width: calc(2ch + 0.25em);
 	}
+	.input-color::part(line) {
```

Comment:
@CarloBar1 Did you tested this ? It doesn't look like this could work as there is no .input-color which has a line part inside the ShadowDOM of  cosmoz-input. 
You'd need something like `:host([type="color"]) .line`.

Reply:
Also, the local server doesn't show all the different kinds of inputs - there seems to be only the text one

Reply:
If its missing lets add a storybook for a input of type color.